go/tools/gopackagesdriver/flatpackage.go

```diff
@@ -213,12 +211,12 @@ func (fp *FlatPackage) ResolveImports(resolve ResolvePkgFunc, overlays map[strin
 			if imp == "C" {
 				continue
 			}
-			if _, ok := fp.Imports[imp]; ok {
+			if pkg := pkg.Imports[imp]; pkg != nil {
```

Comment:
With the default nogo rules, this triggers an issue due to the shadowing:

```
nogo: errors found by nogo during build-time code analysis:
external/rules_go~/go/tools/gopackagesdriver/flatpackage.go:214:7: declaration of "pkg" shadows declaration at line 185 (shadow)
```

Reply:
Thanks. I'll fix this up real quick